exercises/concept/currency-exchange/.docs/instructions.md

```diff
@@ -62,16 +59,13 @@ This function should return the number of bills after exchanging all your money.
 
 ## 5. Calculate value after exchange
 
-Create the `exchangeable_value()` function, with parameter `budget`, `exchange_rate`, `spread`, & `denomination`.
+Create the `exchangeable_value()` function, taking `budget`, `exchange_rate`, `spread`, and `denomination`.
 
-1. `budget` : amount of your money you are planning to exchange.
-2. `exchange_rate` : unit value of the foreign currency.
-3. `spread` : percentage taken as exchange fee.
-4. `denomination` : the value of a single bill.
+Parameter `spread` is the *percentage taken* as exchange fee. If `1.00 EUR == 1.20 USD` and the *spread* is `10%`, the actual exchange will be: `1.00 EUR == 1.32 USD`.
```

Comment:
```suggestion
Parameter `spread` is the *percentage taken* as an exchange fee.
If `1.00 EUR == 1.20 USD` and the *spread* is `10%`, the actual exchange will be: `1.00 EUR == 1.32 USD`.
```

Reply:
When I read `10%` I think `0.10`. It would be nice to clarify that `10%` is conveyed as `10`.

Reply:
I think it's good that we do not tell them, it'll give them something to find out... We cant hand them everything :)